packages/amplify-e2e-tests/src/cleanup-e2e-resources.ts

```diff
@@ -138,6 +154,15 @@ const getOrphanTestIamRoles = async (account: AWSAccountInfo): Promise<IamRoleIn
   return staleRoles.map(it => ({ name: it.RoleName }));
 };
 
+const getOrphanPinpointApplications = async (account: AWSAccountInfo, region: string): Promise<PinpointAppInfo[]> => {
+  const pinpoint = new aws.Pinpoint(getAWSConfig(account, region));
+  const apps = await pinpoint.getApps().promise();
```

Comment:
@lazpavel there are two things to improve here, see https://github.com/aws-amplify/amplify-cli/pull/11393/files#diff-3d06c1a8a2437ab36dd30a667833f2454752b674c0fbeab73823724aa839ebe3R686-R688

1. we need to add us east 1 to region list
2. getApps() returns page of 10 results by default, we should either call with larger page size or follow next token (see https://aws.amazon.com/blogs/developer/pagination-using-async-iterators-in-modular-aws-sdk-for-javascript/ )